added_mass/proposal.md

```diff
@@ -0,0 +1,265 @@
+the body in one direction results in an acceleration of the surrounding fluid in
+a different direction.  [2]
+
+$$
+    \bf{\mu}
+    =
+    \begin{bmatrix}
+      xx & xy & xz & xp & xq & xr \\\
+      xy & yy & yz & yp & yq & yr \\\
+      xz & yz & zz & zp & zq & zr \\\
+      xp & yp & zp & pp & pq & pr \\\
+      xq & yq & zq & pq & qq & qr \\\
+      xr & yr & zr & pr & qr & rr
+    \end{bmatrix}
+$$
+
+where
+
+* \\(xx\\) : added mass in the X direction due to linear acceleration in the X direction
+* \\(xy\\) : added mass in the X direction due to linear acceleration in the Y direction, and vice-versa
+* \\(xz\\) : added mass in the X direction due to linear acceleration in the Z direction, and vice-versa
+* \\(xp\\) : added mass in the X axis due to angular acceleration about the X direction, and vice-versa
+* \\(xq\\) : added mass in the X axis due to angular acceleration about the X direction, and vice-versa
+* \\(xr\\) : added mass orce along the X axis due to angular acceleration about the X direction, and vice-versa
+* \\(yy\\) : added mass in the Y axis due to linear acceleration in the Y direction
+* \\(yz\\) : added mass in the Y axis due to linear acceleration in the Z direction, and vice-versa
+* \\(yp\\) : added mass in the Y axis due to angular acceleration about the X direction, and vice-versa
+* \\(yq\\) : added mass in the Y axis due to angular acceleration about the X direction, and vice-versa
+* \\(yr\\) : added mass in the Y axis due to angular acceleration about the X direction, and vice-versa
+* \\(zz\\) : added mass in the Z axis due to linear acceleration in the Z direction
+* \\(zp\\) : added mass in the Z axis due to angular acceleration about the X direction, and vice-versa
+* \\(zq\\) : added mass in the Z axis due to angular acceleration about the X direction, and vice-versa
+* \\(zr\\) : added mass in the Z axis due to angular acceleration about the X direction, and vice-versa
+* \\(pp\\) : added mass moment about the X axis due to angular acceleration about the X direction
+* \\(pq\\) : added mass moment about the X axis due to angular acceleration about the X direction, and vice-versa
+* \\(pr\\) : added mass moment about the X axis due to angular acceleration about the X direction, and vice-versa
+* \\(qq\\) : added mass moment about the Y axis due to angular acceleration about the X direction
+* \\(qr\\) : added mass moment about the Y axis due to angular acceleration about the X direction, and vice-versa
+* \\(rr\\) : added mass moment about the Z axis due to angular acceleration about the Z direction
+
+## Proposed implementation
+
+While many physics engines may not have direct support for added mass, they often
+support directly setting each element of the resulting \\(\bf{M} + \bf{\mu}\\)
+matrix that's multiplied by acceleration. That's the case for
+[DART](https://dartsim.github.io/dart/main/d6/d91/classdart_1_1dynamics_1_1Inertia.html#aa4661f9950d5958efa4c8609d42aedf7),
```

Comment:
I was just looking more closely at DART's inertia code, and it looks like it will print warnings if a spatial inertia matrix has nonzero values in off-diagonal parts of the translational 3x3 block-diagonal matrix (`xy`, `xz`, `yz`) or the diagonal parts of the block-off-diagonal 3x3 matrix (`xp`, `yq`, `zr`). For rigid bodies, these terms are all `0`, so I think DART will complain a bit about using them for fluid added mass, though it might still work despite the complaints, depending on what assumptions they make when inverting the spatial inertia matrix

Reply:
Yeah that's a concern that @hamilton8415 and I have had from the beginning. Hopefully it's just a matter of suppressing those warnings. It would be nice to have a prototype implementation out soon so we can have an idea of what breaks and what doesn't.

Reply:
From my initial testing, all we need to do is suppress those warnings, see 

* https://github.com/dartsim/dart/pull/1672

I'll check with @hamilton8415 in the coming days if the resulting behavior is what's expected though.